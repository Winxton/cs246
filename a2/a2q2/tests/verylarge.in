5
1
100
200
300
1000
20000001
