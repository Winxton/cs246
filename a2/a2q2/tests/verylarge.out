20000 x 1000
1 x 1
